ui-particles-angular/projects/ui-particles-angular/src/lib/gio-banner/gio-banner.component.scss

```diff
@@ -38,40 +38,64 @@ $types: (
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  padding: 16px;
+  padding: 12px 16px;
+  background-color: $background;
   border-radius: 8px;
 
   @each $typeName, $typePalette in $types {
     &.#{$typeName} {
-      border: 1px solid mat.get-color-from-palette($typePalette, default);
-      background-color: mat.get-color-from-palette($typePalette, lighter);
+      border: 3px solid mat.get-color-from-palette($typePalette, default);
       color: mat.get-color-from-palette($typePalette, darker);
+
+      .mat-icon {
+        color: mat.get-color-from-palette($typePalette, default);
+      }
+
+      ::ng-deep {
+        a {
+          color: mat.get-color-from-palette($typePalette, default);
+        }
+      }
     }
   }
 
   &__wrapper {
+    position: relative;
     display: flex;
     align-items: center;
 
     @include mat.typography-level($typography, body-2);
 
     &__icon {
       display: flex;
+      align-self: flex-start;
       margin-right: 16px;
     }
 
     &__head {
+      display: flex;
       flex: 1 1 auto;
+      flex-direction: column;
       line-height: normal;
     }
-  }
 
-  &__body {
-    padding-top: 2px;
-    padding-left: 24px + 16px;
-    @include mat.typography-level($typography, body-1);
+    &__title {
+      display: flex;
+      flex: 1;
+      align-items: center;
+      padding: 4px 0;
+    }
+
+    &__action {
+      align-self: flex-start;
```

Comment:
the button does not centre and remain aligned at the top? 
![image](https://user-images.githubusercontent.com/4974420/203985911-e509b9ca-3d17-49f1-a076-c9b9cec79619.png)